packages/size-limit/test/get-config.test.js

```diff
@@ -188,10 +188,8 @@ it('normalizes import', async () => {
     checks: [
       {
         import: {
-          [fixture(
-            'integration-esm',
-            'index.js'
-          )]: '{ VERY_LONG_NAME_FOR_CONST_TO_TEST_TREE_SHAKING }'
+          [fixture('integration-esm', 'index.js')]:
```

Comment:
I have touched this file once and the git hook keeps wanting to reformat it although there are no changes.